images/hub/jupyterhub_config.py

```diff
@@ -290,27 +371,25 @@
     })
 
 for name, service in get_config('hub.services', {}).items():
-    api_token = get_secret('services.token.%s' % name)
     # jupyterhub.services is a list of dicts, but
     # in the helm chart it is a dict of dicts for easier merged-config
     service.setdefault('name', name)
+    # handle camelCase->snake_case of api_token
+    api_token = service.pop('apiToken', None)
     if api_token:
         service['api_token'] = api_token
     c.JupyterHub.services.append(service)
```

Comment:
@minrk I don't see how the `api_token` propegates to here without `get_secret` that utilized a path where `hub-secret` is mounted, which is different from where the `hub-config` is mounted.

Reply:
Figured it out, you are merging the two mounted directories, nice i like it!